lib/Remotery.c

```diff
@@ -1012,6 +1012,45 @@ int ashmem_create_region(const char* name, size_t size)
     close(fd);
     return ret;
 }
+
+// https://chromium.googlesource.com/chromium/src/+/HEAD/third_party/ashmem/ashmem-dev.c
```

Comment:
As seen in Chromium.
Tested on Android 8 and Android 11

Reply:
Can we have a copy of the license here as well? AFAIA the Apache License 2.0 requires it.

Reply:
Also, use the following link so that it persists if `HEAD` is changed and it links directly to the line: https://chromium.googlesource.com/chromium/src/+/ad02487d87120bd66045960ffafe0fc27600af50/third_party/ashmem/ashmem-dev.c#181